plugins/generator-1.18/forge-1.18/utils/triggers.java.ftl

```diff
@@ -0,0 +1,398 @@
+<#include "procedures.java.ftl">

```

Comment:
If all these triggers were not tested, please only include the ones you tested and use

Reply:
I tested them all (all item triggers)